server/server.js

```diff
@@ -327,6 +327,9 @@ app.post(
   keycloak.getUserInfoMiddleware(),
   asyncMiddleware(async (req, res) => {
     await validate(ParticipantStatusChange, req.body);
+    if (req.body.status === 'archived') {
```

Comment:
I always suggest immutability of req object
`const status = req.body.status === 'archive' ? 'rejected' : req.body.status`

Reply:
I ended up removing this part and just adding a new status